### PR TITLE
[luci-interpreter]Add L2Normalize Kernel

### PR DESCRIPTION
**Pull request by struss**
This commit add L2Normalize Kernel on luci-interpreter.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

File: compiler/luci-interpreter/src/kernels/L2Normalize.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  L2Normalize(const Tensor *input, Tensor *output, const L2NormParams &params);
+
+  std::vector<const Tensor *> getInputTensors() const override { return {_input}; }
+  std::vector<Tensor *> getOutputTensors() const override { return {_output}; }
+
+  void configure() override;
+  void execute() const override;
+
+private:
+  inline void evalFloat() const;
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/L2Normalize.h:40]: `inline` keyword is useless here.

File: compiler/luci-interpreter/src/kernels/L2Normalize.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      break;
+    case DataType::U8:
+      evalQuantized();
+      break;
+    default:
+      throw std::runtime_error("Unsupported type.");
+  }
+}
+
+inline void L2Normalize::evalFloat() const
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/L2Normalize.cpp:64]: `inline` keyword is useless here.
What I meant by "inlining" in my previous comment is incorporating the function body into its caller (optional).
Please remove the keyword or inline the functions.

Comment (struss) [reply compiler/luci-interpreter/src/kernels/L2Normalize.cpp:64]: Oh, Sorry. I will make a Optional change. Thanks for telling me that problem.

File: compiler/luci-interpreter/src/kernels/L2Normalize.h (hunk 1 above)
Comment (struss) [reply compiler/luci-interpreter/src/kernels/L2Normalize.h:40]: Thanks, I will remove this.

**Pull request merged**